config/ext_jenkins-release.yaml

```diff
@@ -264,129 +264,6 @@ controller:
                                 operator: In
                                 values:
                                   - windows
-            - amazonEC2:
-                cloudName: "aws-us-east-2"
-                credentialsId: "ec2-agents-credentials"
```

Comment:
Do you mind also removing the credentials (earlier in this file) as part of this PR, as it is not used anywhere else?

Reply:
Of course, done in [`b25d151` (#3774)](https://github.com/jenkins-infra/kubernetes-management/pull/3774/commits/b25d15175ccc15b630dfb9dd18eb960b4e584245)